src/Response/ErrorResponse.php

```diff
@@ -39,18 +33,14 @@ public static function fromArray(array $response): self
             throw new \RuntimeException('Cannot create ErrorResponse');
         }
 
-        $response = $response['error'];
-
-        Assert::keyExists($response, 'message');
-        Assert::keyExists($response, 'id');
-        Assert::keyExists($response, 'code');
+        Assert::keyExists($response, 'error');
+        Assert::keyExists($response, 'key');
+        Assert::keyExists($response, 'status');
```

Comment:
if you cast status onto int check `isNumeric` as well

Reply:
Ok